src/sentry/conf/server.py

```diff
@@ -1067,8 +1031,6 @@ def create_partitioned_queues(name):
     "organizations:invite-members": True,
     # Enable rate limits for inviting members.
     "organizations:invite-members-rate-limits": True,
-    # Enable removing issue from issue list if action taken.
-    "organizations:issue-list-removal-action": False,
```

Comment:
@scttcper Was this a feature we were planning on adding soon? Wasn't sure if it was related to that pr from last week?

Reply:
good catch, this is a flag we're about to use